OneCommunityGlobal/HGNRest
# Yongjian - Update message for inactive user account after login

# Description
![Screen Shot 2023-07-07 at 11 55 04 AM](https://github.com/OneCommunityGlobal/HGNRest/assets/46613683/1b3e8782-9761-4cf4-8a69-8c57c66189af)

## Main changes explained:
Display the message "Sorry, this account is no longer active. If you feel this is in error, please contact your Manager and/or Administrator." if the user exists in the database but his or her profile is not active.

## How to test:
1. check into current branch from the backend
2. do `npm run build` and `npm start` from the backend
3. check into development branch from the frontend
4. do `npm run start:local` to run the frontend
5. log in as admin user
6. go to dashboard→ User Management -> search for a non-admin role of under your account -> change "Active" status to "Inactive" by clicking on the green dot
7. log out of your admin account
8. attempt to login with the non-admin role with status already set to "Inactive" from step 6
9. verify that the message "Sorry, this account is no longer active. If you feel this is in error, please contact your Manager and/or Administrator." is displayed after clicking on submit button after entering credentials

## Screenshots
![Screen Shot 2023-07-07 at 11 49 27 AM](https://github.com/OneCommunityGlobal/HGNRest/assets/46613683/2f104715-680e-46da-8d37-64b0a1fbfc68)

![Screen Shot 2023-07-07 at 11 47 18 AM](https://github.com/OneCommunityGlobal/HGNRest/assets/46613683/a8a33955-3540-4845-9c9a-adf7fed757e8)

![Screen Shot 2023-07-07 at 11 47 25 AM](https://github.com/OneCommunityGlobal/HGNRest/assets/46613683/871ec0f6-b28b-4db2-b2b4-0e95c33e57fb)
